README.md

```diff
@@ -43,45 +43,47 @@ npm run build
 npm run lint
 ```
 
-### How to start the repository server
+### How to start the LionWeb server
 Ensure that Postgres is running.
-The repository server is started with `npm run dev-run` in  the `packages/server` folder:
+The server is started with `npm run dev-run` in  the `packages/server` folder:
 
 ```
 cd packages/server
 npm run dev-run
 ```
 
-For more information on how to configure the repository, please check [configuration.md](configuration.md).
+For more information on how to configure the server, please check [configuration.md](configuration.md).
 
 ### How to test
-Ensure the Postgres server and the repository server are both running.
+Ensure the Postgres server and the LionWeb server are both running.
 Then do
 
 ```
 npm run test
 ```
 
 ## Status
-This repository is Work In Progress, currently:
+This server is Work In Progress, currently:
```

Comment:
I think that it is starting to be reasonably mature, so I would tone down this sentence